stats/stats_test.go

```diff
@@ -81,6 +83,52 @@ var (
 	}
 	// The id for which the service handler should return error.
 	errorID int32 = 32202
+    // Server Stats
+	// Ensure that Unary RPC server stats events are logged in the correct order.
```

Comment:
variable name is self explanatory. If at all we want to put comment here, something like this is more suitable, `To verify if the Unary RPC server stats events are logged in the correct order.`